definitions/ext-ups/kentik-apc_ups-dashboard.json

```diff
@@ -2,195 +2,221 @@
-						"row": 4,
-						"height": 3,
-						"width": 4
-					},
-					"title": "Output Load Capacity %",
-					"rawConfiguration": {
-						"facet": {
-							"showOtherSeries": false
-						},
-						"legend": {
-							"enabled": true
-						},
-						"nrqlQueries": [
-							{
-								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.upsAdvOutputLoad) AS 'Output Load %' WHERE provider = 'kentik-ups' TIMESERIES 5 MINUTES"
-							}
-						],
-						"yAxisLeft": {
-							"max": 100,
-							"min": 0,
-							"zero": false
-						}
-					}
-				},
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT if(latest(upsAdvBatteryReplaceIndicator) = 'noBatteryNeedsReplacing', 0, 1) FACET upsAdvBatteryReplaceIndicator WHERE provider = 'kentik-ups'"
+				}
+			  ],
+			  "platformOptions": {
+				"ignoreTimeRange": false
+			  },
+			  "thresholds": [
 				{
-					"visualization": {
-						"id": "viz.billboard"
-					},
-					"layout": {
-						"column": 5,
-						"row": 4,
-						"height": 3,
-						"width": 2
-					},
-					"title": "Battery Needs Replacing (1 == All Clear, 2 == Needs Replacing)",
-					"rawConfiguration": {
-						"dataFormatters": [],
-						"nrqlQueries": [
-							{
-								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.upsAdvBatteryReplaceIndicator) AS 'Battery Needs Replacing' WHERE provider = 'kentik-ups'"
-							}
-						],
-						"thresholds": [
-							{
-								"alertSeverity": "WARNING",
-								"value": 1.1
-							},
-							{
-								"alertSeverity": "CRITICAL",
-								"value": 1.9
-							}
-						]
-					}
+				  "alertSeverity": "WARNING",
+				  "value": 0.1
 				},
 				{
-					"visualization": {
-						"id": "viz.billboard"
-					},
-					"layout": {
-						"column": 7,
-						"row": 4,
-						"height": 3,
-						"width": 2
-					},
-					"title": "Bad Battery Packs (Count)",
-					"rawConfiguration": {
-						"dataFormatters": [],
-						"nrqlQueries": [
-							{
-								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.upsAdvBatteryNumOfBadBattPacks) AS 'Bad Battery Packs' WHERE provider = 'kentik-ups'"
-							}
-						],
-						"thresholds": [
-							{
-								"alertSeverity": "WARNING",
-								"value": 0.5
-							},
-							{
-								"alertSeverity": "CRITICAL",
-								"value": 0.9
-							}
-						]
-					}
+				  "alertSeverity": "CRITICAL",
+				  "value": 0.2
+				}
+			  ]
+			}
+		  },
+		  {
+			"title": "Output Load Capacity %",
+			"layout": {
+			  "column": 5,
+			  "row": 1,
+			  "width": 8,
+			  "height": 4
+			},
+			"visualization": {
+			  "id": "viz.line"
+			},
+			"rawConfiguration": {
+			  "facet": {
+				"showOtherSeries": false
+			  },
+			  "legend": {
+				"enabled": true
+			  },
+			  "nrqlQueries": [
+				{
+				  "accountId": 0,
+				  "query": "FROM Metric SELECT latest(kentik.snmp.upsAdvOutputLoad) AS 'Output Load %' WHERE provider = 'kentik-ups' TIMESERIES 5 MINUTES"
```

Comment:
I would remove the 5 MINUTES from timeseries or set it to MAX as timerange can be changed by user

Reply:
Thanks @thezackm, and then if the user changes the timerange to one day, or one week it's correctly adapted the timesries?

Reply:
correct; the UI accepts the change and smooths the line out; we've been using this pattern for over a year on all network entities